### DISCUSSION TITLE
大家好！绳网新人一枚˙ᗜ˙

### DISCUSSION THREAD

**Discussion by yuhuiou** [share121/inter-knot#260] [其他]
大家好！
![IMG_20240809_232723](https://github.com/user-attachments/assets/20e35a18-0f86-4a86-a68b-aac03446e851)

**Comment by KinoZR** [comment]
你觉得我好吗？差不多得了，新人一进来就这么狂。。

**Comment by lllliufeng** [comment]
楼上的，对新人好点

**Comment by Feng-shao** [comment]
你好
